datenguide_python/query_builder.py

```diff
@@ -351,8 +523,41 @@ def get_fields(self) -> List[str]:
         """
         return self.start_field.get_fields()
 
-    def results(self) -> Optional[ExecutionResults]:
-        return QueryExecutioner().run_query(self)
+    def results(self) -> Union[DataFrame, Optional[ExecutionResults]]:
```

Comment:
I don't understand the ExectutionResults part of the signature.

Reply:
i just copied the return type from the transform function of the Transformer. But apparently this has changed :D